trio/_core/_run.py

```diff
@@ -1643,10 +1776,10 @@ async def python_wrapper(orig_coro):
             self.instruments.call("task_spawned", task)
         # Special case: normally next_send should be an Outcome, but for the
         # very first send we have to send a literal unboxed None.
-        self.reschedule(task, None)
+        self.reschedule(task, None)  # type: ignore[arg-type]
```

Comment:
I know it's a special case but why not just have reschedule's annotation accept `None`?

Reply:
Well that's exposed as a public API, so I didn't want to add the special case there too.